stubs/AMQP.php

```diff
@@ -66,11 +66,16 @@
 
 /**
  * When publishing a message, the message must be routed to a valid queue. If it is not, an error will be returned.
+ * Note: This flag does not currently work in pecl-amqp, and WILL cause undefined behavior if you use it.
+ * @link https://bugs.php.net/bug.php?id=62786
```

Comment:
Are you sure about that? RabbitMQ supports the MANDATORY flag (https://www.rabbitmq.com/specification.html) but if you look in #23 the problem seems to be that the underlying C library used by php-amqp uses a callback here, which doesn't work well with the synchronous way PHP (and thus php-amqp) is implemented. In other words, (although I haven't tried with other brokers) the problem here is in php-amqp, not RabbitMQ.